docs/doc_sources/api_reference/dpctl/tensor.rst

```diff
@@ -0,0 +1,50 @@
+.. _dpctl_tensor_pyapi:
+
+:py:mod:`dpctl.tensor`
+======================
+
+.. py:module:: dpctl.tensor
+
+.. currentmodule:: dpctl.tensor
+
+:py:mod:`dpctl.tensor` provides a reference implementation of
+:array_api:`Python Array API <>` specification. The implementation
+uses :ref:`data-parallel <parallelism_definitions>` algorithms suitable for execution on accelerators,
+such as GPUs.
+
+:py:mod:`dpctl.tensor` is written using C++ and :sycl_spec_2020:`SYCL <>`
+and oneAPI extensions implemented in :dpcpp_compiler:`Intel(R) oneAPI DPC++ compiler <>`.
+
+This module contains:
```

Comment:
Some undocumented things:

- all constants (`pi`, `e`, `nan`, `inf`, and `newaxis`)
- `asnumpy` and `to_numpy`
- `dtype` itself
- in elementwise functions, no documentation is rendered for `nin`, `nout`, `types`, etc. special attributes
- Print utilities do not have rendered documentation
- Flags object on `usm_ndarray` is mentioned but has no dedicated page for documenting various flags
- Order in general (C-contiguous vs. F-contiguous) may be worth a quick note